src/libtorch.cc

```diff
@@ -122,6 +126,12 @@ class ModelState : public BackendModel {
   // Flag pair to indicate whether nvfuser is set and enabled respectively.
   // Defaults to (false, false).
   std::pair<bool, bool> enable_nvfuser_pair_;
+
+  // Share torch module across instances on the same device. They key is a pair
```

Comment:
Share TorchScript model across all instances on the same device. The key is a pair of isGPU and device index.